config.py

```diff
@@ -75,6 +79,10 @@ class DevelopmentConfig(Config):
     SECURITY_USER_PASSWORD = os.getenv('SECURITY_USER_PASSWORD', 'secret')
     BASIC_AUTH = (SECURITY_USER_NAME, SECURITY_USER_PASSWORD)
 
+    PARTY_SERVICE_URL = os.getenv('PARTY_SERVICE_URL', 'http://localhost:8081')
+    PARTY_SERVICE_USERNAME = os.getenv('PARTY_SERVICE_USERNAME', 'admin')
+    PARTY_SERVICE_PASSWORD = os.getenv('PARTY_SERVICE_PASSWORD', 'secret')
```

Comment:
Are we going to have different username and password for every service? Might be useful to have PARTY_BASIC_AUTH = (PARTY_SERVICE_USERNAME, PARTY_SERVICE_PASSWORD) like how we currently use BASIC_AUTH. (or PARTY_SERVICE_BASIC_AUTH once we agree on config naming standards...)